plugin.video.orange.fr/LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) BreizhReloaded.
+Copyright (c) François Lavaud-Wernert.
```

Comment:
Why the name change? It seems like the commit/maintainer did not change?

Reply:
I just thought that it would be cleaner to use my real name on my projects and I noticed that I forgot to update the licence file. I switched my GitHub account from [BreizhReloaded ](https://github.com/BreizhReloaded/) to [Flawe](https://github.com/f-lawe/) and moved all my repos.